kotlin-react/src/main/kotlin/react/experimental/createElement.kt

```diff
@@ -0,0 +1,20 @@
+package react.experimental
+
+import kotlinext.js.ReadonlyArray
+import react.*
+
+fun createElement(
+    block: Render,
+): ReactElement? {
+    val children: ReadonlyArray<ReactNode>? = ChildrenBuilder()
+        .apply(block)
+        .asDynamic()
```

Comment:
it seems `asDynamic` can be omitted

Reply:
No `children` property in `ChildrenBuilder`

Reply:
If so, what is it?
```
private inline var ChildrenBuilder.children: ReadonlyArray<ReactNode>?
```

Reply:
Fixed